api-reference/10 UI Components/dxNumberBox/1 Configuration/format.md

```diff
@@ -8,13 +8,115 @@ default: ''
 Specifies the value's display format and controls user input accordingly.
 
 ---
-[note] With this property specified, a press on Minus Sign (-) inverts the current value instead of entering "-".
-
-[note] If you set this property, the telephone keyboard is used for editing on mobile devices. However, it may not have a point, comma, or other symbols for entering decimals. Set the [mode](/api-reference/10%20UI%20Components/dxNumberBox/1%20Configuration/mode.md '/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#mode') property to *"text"* to use the standard keyboard instead.
-
 #include common-demobutton with {
     url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/NumberBox/Formatting/"
 }
 
+The format property can accept the following value types:
+
+- Predefined format    
+
+        <!-- tab: JavaScript -->
+        const number = 1.2345;
+
+        type: "fixedPoint" // 1
+        type: "decimal" // 1.2345
+        type: "percent" // 123%
+        type: "currency" // $1
+
+    You can specify the [precision](/Documentation/ApiReference/Common/Object_Structures/format/#precision) property for decimal numbers.
+
+    NumberBox does not support large number formats. If you want to apply these formats, you can implement the [formatter](/api-reference/50%20Common/Object%20Structures/format/formatter.md '/Documentation/ApiReference/Common/Object_Structures/format/#formatter') and [parser](/api-reference/50%20Common/Object%20Structures/format/parser.md '/Documentation/ApiReference/Common/Object_Structures/format/#parser') functions or specify a custom format string.
+
+- Custom format string    
+
+    Use Unicode Locale Data Markup Language (LDML) patterns to specify a custom format string. An LDML pattern consists of wildcard characters and characters displayed as is. The **format** property supports the following wildcard characters:
+
+    <table class="dx-table">
+        <tr>
+        <th>Format character</th>
+        <th>Description</th>
+        </tr>
+        <tr>
+        <td>0</td>
+        <td>A digit. Displays '0' if it is not specified in the UI.</td>
+        </tr>
+        <tr>
+        <td>&#35;</td>
+        <td>
+                A digit or nothing. One symbol represents several integer digits, but only one decimal digit.</br> 
```

Comment:
Adding a few details. 

```suggestion
                Any number of leading digits, a single digit, or nothing. If this character goes first in the format string, it can match multiple leading digits (before the decimal point). Consequent characters match a single digit. If the formatted number doesn't have a digit in the corresponding position, displays nothing.</br> 
```